.gitignore

```diff
@@ -24,6 +24,7 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
+client_secrets.json
```

Comment:
client_secrets.json is already listed in the gitignore

Reply:
removed